### ISSUE TITLE
Use black font on the header for supervisor contacts

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1827]
There's not enough contrast between the yellow and white.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/4ea6129a-27c8-4ce5-af86-6410e1766816)

**Comment by mcharg** [comment]
Looks good.
